lit/docs/install/worker.lit

```diff
@@ -211,7 +214,7 @@ decide much on its own.
     When a worker machine is going away, it should be \italic{retired}. This is
     similar to \italic{landing}, except at the end the worker is completely
     unregistered, along with its volumes and containers. This should be done
-    when a worker's VM or container is being destroyed.
+    when a worker's VM or container are being destroyed.
```

Comment:
I think the `is` is appropriate here - it's referring to the worker VM/container itself which is singular